aptos-move/block-executor/src/txn_last_input_output.rs

```diff
@@ -448,16 +441,12 @@ impl<T: Transaction, O: TransactionOutput<Txn = T>, E: Debug + Send + Clone>
 
     // Must be executed after parallel execution is done, grabs outputs. Will panic if
     // other outstanding references to the recorded outputs exist.
-    pub(crate) fn take_output(
-        &self,
-        txn_idx: TxnIndex,
-    ) -> ExecutionStatus<O, BlockExecutionError<E>> {
+    pub(crate) fn take_output(&self, txn_idx: TxnIndex) -> ExecutionStatus<O, E> {
```

Comment:
Did we change the error type from BlockExecutionError<E> to E because  BlockExecutionError<E> is the error type for a whole block and not for a single transaction?

Reply:
we used to wrap status, after recent refactorings we can simplify.